webpack.config.js

```diff
@@ -0,0 +1,18 @@
+const path = require('path');
+
```

Comment:
### THIS IS JUST TO MAKE THE UI SHOW, NOT ALL THE CORRECT SETUP. YOU SHOULD TRY AND REVISIT THE WEBPACK TUTORIAL TO SET IT UP CORRECTLY

- Add this HTML plugin 
`const HtmlWebpackPlugin = require('html-webpack-plugin');`

- Install the `npm install html-webpack-plugin` 

*Incase you have a linter error from the `webpack.config.js` file, add this eslint disable at the top of the file.* 
`/* eslint-disable */`